test/signed-off-by-test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+first_signed_off_commit="d9c847d8a71f93593d59ec48372f7fa80f49fd08"
+success="true"
+
+for commit in $(git log --pretty=oneline $first_signed_off_commit..HEAD | cut -d " " -f 1)
+do
+    if $(git show $commit | grep -q Signed-off-by ); then
+        :
+    else
+        echo "Commit $commit doesn't have a Signed-off-by"
+        git show $commit
+        success="false"
+    fi
+
+done
+
+if [ "$success" = "true" ]; then
+    echo "All commits have a Signed-off-by"
+else
+    echo "Some commits do not have a signed off by, failing"
```

Comment:
It would be better as Signed-off-by for consistency.

Maybe even 'Signed-off-by (git commit -s)' instead, so that it's extremely obvious to new contributors.